code/modules/power/singularity/narsie.dm

```diff
@@ -255,7 +255,7 @@
 ///Called only if the crew managed to destroy narsie at the very last second for [/proc/begin_the_end()]
 /proc/narsie_last_second_win()
 	set_security_level("red")
-	SSshuttle.clearHostileEnvironment()
+	SSshuttle.clearHostileEnvironment(GLOB.cult_narsie)
```

Comment:
Isn't `GLOB.cult_narsie` null at this point? Tested?

Reply:
Maybe, I'll test at some point (no I didn't forget about this PR what do you mean :))

Reply:
Yeah you're right, it was already nulled, I threw the clear environment before it gets nulled.

Though that might be a bad way to do it...